compiler/src/iree/compiler/Dialect/Stream/Transforms/DumpStatistics.cpp

```diff
@@ -293,9 +293,9 @@ static void prettyPrintStreamInfo(const UsageInfo &usageInfo,
                                   llvm::raw_fd_ostream &os) {
   auto parentOp = executeOp->getParentOfType<mlir::FunctionOpInterface>();
 
-  prettyPrintItemHeader(
-      llvm::formatv("stream.cmd.execute", parentOp->getName().getStringRef()),
-      os);
+  prettyPrintItemHeader(llvm::formatv(false, "stream.cmd.execute",
+                                      parentOp->getName().getStringRef()),
+                        os);
```

Comment:
What is this supposed to do?

Reply:
Don't use formatv just to concatenate strings

Reply:
Its becuase after https://github.com/llvm/llvm-project/pull/105745, uses of formatv by default have validation to check certain things such as uses having the same number of arguments that the the input string is expecting. However, it is not always possible to know this at compile time, so they give an option to turn off the validation, so that is the `false` argument.

Reply:
I dont think this is concatenating strings, I am fairly sure the second argument ` parentOp->getName().getStringRef()` is just getting ignored. Anyway outside of scope of integrate to fix this kind of things. with the `false` it goes back to the same behavior.